packages/examples/src/security/oauth/consumer.ts

```diff
@@ -18,7 +18,7 @@ import { Helpers } from "@node-wot/core";
 let WoT: WoT.WoT;
 let WoTHelpers: Helpers;
 
-WoTHelpers.fetch("https://localhost:8080/OAuth").then(td => {
+WoTHelpers.fetch("https://localhost:8080/oauth").then(td => {
```

Comment:
Probably this should be changed also in the root example folder.

Reply:
I think I already did it if you mean `examples/security/oauth/consumer.js` file.